src/systems/physics-system.js

```diff
@@ -217,35 +213,39 @@ export class PhysicsSystem {
 
   addBody(object3D, options) {
     this.workerHelpers.addBody(this.nextBodyUuid, object3D, options);
-    this.object3Ds[this.nextBodyUuid] = object3D;
-    this.bodyOptions[this.nextBodyUuid] = options;
-    this.collisions[this.nextBodyUuid] = [];
-    this.bodyLinearVelocities[this.nextBodyUuid] = 0;
-    this.bodyAngularVelocities[this.nextBodyUuid] = 0;
+
+    this.bodies.set(this.nextBodyUuid, {
```

Comment:
Same as above -- maybe a name like `bodyInfos` or `bodyInfoForUuid` to indicate that it's a collection of stuff related to this body uuid